python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,833 @@
+      "text/plain": [
+       "<Figure size 864x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# defining and plotting request/path to natural_trees.txt?\n",
+    "def area():\n",
+    "    filter='(landuse=forest or natural=wood) and geometry:polygon'\n",
+    "    res = elements(\"/area/density/groupBy/boundary\"\n",
+    "         ,filter=filter\n",
+    "         ,bpolys=regions\n",
+    "         ,time=t_mon)\n",
+    "    display(res.url)\n",
+    "    green = res.json()\n",
+    "    #display(body)\n",
+    "    df = pd.DataFrame()\n",
+    "    for groupBy in green['groupByResult']:\n",
+    "        series = pd.Series({row['timestamp']:row['value'] for row in groupBy['result']})\n",
+    "        df[groupBy['groupByObject']] = series\n",
+    "    df.set_index(pd.to_datetime(df.index),inplace=True)\n",
+    "    df.to_pickle(\"./df_density.pkl\")\n",
+    "    df.plot(figsize=(12,10), fontsize=11)\n",
+    "area()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "terminal-treasury",
+   "metadata": {},
+   "source": [
+    "#### _In the graphic above you can see the development of the density of natural=wood/landuse=forest information_"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "editorial-scotland",
+   "metadata": {},
+   "source": [
+    "When looking at the plot of the output dataset it is pretty safe to say that Edmonton has the lowest density of forest/wood information given when comparing it to the other regions. Although this might be part of low mapping activities too, it might as well be related to local oil and gas production areas or farmland which reduced the amount of forest. Another factor could be the general geographic connection to the Prairies Ecozone.\n",
+    "\n",
+    "Furthermore it is worth mentioning that all cities, except Vancouver, only really added their forest information around 2010 and 2011 (Edmonton). \n",
+    "Interestingly enough the density values for Vancouver, though starting to increase the earliest, remain quite low for the period between 2011-2013 and only cross parts with the Ottawa- and Halifax-values around 2017. Additionally there is a strong increase in density in the early 2013 months. During that same year the Open Data Hackathon took place on Feb23rd as well as the Toronto hack weekend which might be related to the jump or maybe have motivated it (this is merely an assumption).\n",
+    "\n",
+    "Ottawa and Halifax both begin hold increasing information starting around the same period in 2010 and cross each other several times until Ottawa finally takes its place at the top in 2018. Density wise it (Ottawa) is the winner looking at the overall development of density throughout the years. However, it has a short collapse where it decreases in density between 2016 and 2017. _(potential reasons?)_\n",
```

Comment:
I don't understand this first sentence here.

Reply:
me neither, I've corrected it